From dc2234f4062cb772e67d9b02be18a834538d1976 Mon Sep 17 00:00:00 2001
From: Terrence Pu <terrence.pu@amlogic.com>
Date: Mon, 8 Jun 2020 10:31:30 -0700
Subject: [PATCH 6/9] u-boot: Fix A/B update from Firmware Control is not
 alternating

PD#SWPL-27570

Problem:
Thunder Plugin: Firmware Control is not able to update A/B partition correctly

Solution:
Fixed the configuration issue on bootloader for RDK

Verify:
u212/HP40A

Change-Id: I546fe9d3bd955466b35fbcf5b13db28cbc3a505d
Signed-off-by: Terrence Pu <terrence.pug@amlogic.com>
---
 board/amlogic/configs/g12a_u212_v1.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/amlogic/configs/g12a_u212_v1.h b/board/amlogic/configs/g12a_u212_v1.h
index 68d19acb15..e2dd892998 100644
--- a/board/amlogic/configs/g12a_u212_v1.h
+++ b/board/amlogic/configs/g12a_u212_v1.h
@@ -67,7 +67,7 @@
   */
 #define CONFIG_BOOTLOADER_CONTROL_BLOCK
 
-#define CONFIG_CMD_BOOTCTOL_AVB
+//#define CONFIG_CMD_BOOTCTOL_AVB
 
 /* support ext4*/
 #define CONFIG_CMD_EXT4 1
-- 
2.31.0

